### PR TITLE
Update push and test flow to include release branches

### PR DESCRIPTION
**Pull request by bianpengyuan**
Also duplicate wasm files with a more readable name for doc and configuration.

### CODE REVIEW

**Comment by bianpengyuan** [comment]
ping @kyessenov @mandarjog can I get a review on this?

File: .github/workflows/push.yaml [hunk 1]
```diff
@@ -47,7 +49,11 @@ jobs:
             tmp=${WASM_PATH%%.wasm}
             EXTENSION_NAME=$(basename ${tmp})
             mkdir -p wasm-extensions/${EXTENSION_NAME//_/-}
-            mv bazel-bin/${WASM_PATH} wasm-extensions/${EXTENSION_NAME//_/-}/${SHA}.wasm
+            cp bazel-bin/${WASM_PATH} wasm-extensions/${EXTENSION_NAME//_/-}/${GITHUB_SHA}.wasm
+
+            # also push binary with more readable name, so that it could be easily used in configuration and documentation.
+            cp wasm-extensions/${EXTENSION_NAME//_/-}/${GITHUB_SHA}.wasm wasm-extensions/${EXTENSION_NAME//_/-}/${VERSION}-$(date +"%Y-%m-%d-%H-%M-%S").wasm
```
Comment (mandarjog) [inline .github/workflows/push.yaml:55]: Please add example before and after names.
basic_auth-1.7-$(date) I think

**Review by mandarjog — APPROVED** [review]
Lgtm sans comment.

File: .github/workflows/push.yaml (hunk 1 above)
Comment (bianpengyuan) [reply .github/workflows/push.yaml:56]: Done.

**Pull request merged**